packages/tldraw/src/lib/ui/components/SharePanel/DefaultSharePanel.tsx

```diff
@@ -0,0 +1,10 @@
+import { PeopleMenu } from './PeopleMenu'
+
+/** @public @react */
```

Comment:
`@react` is that doing anything?

Reply:
that tags this export as a react component for our documentation system which:
- puts it in a category with other react components
- pulls the documentation for the props and renders it on the same page